client/src/components/Checkout/CheckoutForm/CheckoutForm.tsx

```diff
@@ -0,0 +1,81 @@
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Box from '@material-ui/core/Box';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { CircularProgress } from '@material-ui/core';
+
+interface Props {
+  handleSubmit: (
+    {
+      priceId,
+      quantity,
+    }: {
+      priceId: string;
+      quantity: number;
+    },
+    {
+      setStatus,
+      setSubmitting,
+    }: FormikHelpers<{
+      priceId: string;
+      quantity: number;
+    }>,
+  ) => void;
+}
+
+export default function Checkout({ handleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+
+  if (process.env.REACT_APP_PRICE_ID != undefined) {
```

Comment:
Prefer to use strict equality operator `===`, `!==`, as all falsy values will always be equal when compared using `==` rather than `===`. Or simply test for falsiness with `if (process.env.REACT_APP_PRICE_ID)`

Reply:
Yes, if(object) can check both null and undefined. Thank you for your reminder.